Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Model/Model.h

```diff
@@ -37,6 +37,10 @@ namespace AZ
 
             static Data::Instance<Model> FindOrCreate(const Data::Asset<ModelAsset>& modelAsset);
 
+            //! Orphan the model, its lods, and all their buffers so that they can be replaced in the instance database
+            //! This is a temporary function, that will be removed once the Model/ModelAsset classes no longer need it
+            static void TEMPOrphanFromDatabase(const Data::Asset<ModelAsset>& modelAsset);
```

Comment:
It certainly could stick around for longer than desired, but the goal is to make sure no one starts depending on this functionality so it can be removed when the time does come.